packages/cactus-plugin-ledger-connector-quorum/src/test/resources/network-files/member1/accountkey

```diff
@@ -0,0 +1 @@
+{"address":"f0e2db6c8dc6c681bb5d6ad121a107f300e9b2b5","crypto":{"cipher":"aes-128-ctr","ciphertext":"f2af258ee3733513333652be19197ae7eace4b5e79a346cf25b02a857e6043f3","cipherparams":{"iv":"587d7faaa6403b8a73273d0ad58dd71f"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"b93c7d69c5bb0a760c3b7fdf791c47896a552c5c977648b392a24d708674dcf3"},"mac":"d83bcb555c92fc5a32ceacabbb6b99f59515ec3986b9fe5995c67e027bd750c8"},"id":"5392d73f-08dd-42b8-bca9-6f6d35c419d9","version":3}
```

Comment:
@travis-payne Is this folder necessary to build the container image or does the container image already contain these files? If they are redundant, please remove them, we should have as little as possible hardcoded credentials in the source code even for test purposes. (nit pick, sorry)

The above question applies to the whole folder `member1` not just this file,but there's no way to comment on folders.

Reply:
Fixed as per other comment.